pkgs/os-specific/linux/tuxedo-keyboard/default.nix

```diff
@@ -1,14 +1,14 @@
-{ lib, stdenv, fetchFromGitHub, kernel, kmod }:
+{ lib, stdenv, fetchFromGitHub, kernel }:
 
 stdenv.mkDerivation rec {
   pname = "tuxedo-keyboard-${kernel.version}";
-  version = "2019-08-26";
+  version = "3.0.5";
 
   src = fetchFromGitHub {
     owner = "tuxedocomputers";
     repo = "tuxedo-keyboard";
-    rev = "d65e76e84cfd8169591fc2a0a7c9219fa19da1b5";
-    sha256 = "1s48qpwybwh5pwqas2d1v2a7x4r97sm4hr9i4902r1d7h384bv17";
+    rev = "${version}";
```

Comment:
AFAIU this should be `v${version}`. FWIW it's currently failing to download the tarball for me:

```
λ ma27 [~/Projects/nixpkgs] at  tuxedo-keyboard-update ✔
→ b linuxPackages.tuxedo-keyboard                                                           [a872c3b0f2b]
source> trying https://github.com/tuxedocomputers/tuxedo-keyboard/archive/3.0.5.tar.gz
source>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
source>                                  Dload  Upload   Total   Spent    Left  Speed
source> 100   138  100   138    0     0    552      0 --:--:-- --:--:-- --:--:--   552
source>   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
source> curl: (22) The requested URL returned error: 404
source> error: cannot download source from any mirror
error: builder for '/nix/store/1z88pcvb6vzx0r4xzwfsndls8f7jaxkf-source.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/7gjzybhbrxx2yd73nch13j5rviw0zm4j-tuxedo-keyboard-5.10.32-3.0.5.drv' failed to build

```

Reply:
:facepalm: thanks! :)